src/mapping.ts

```diff
@@ -359,6 +362,7 @@ export function handleNftFeedUpdate(call: UpdateCall): void {
   loan.save()
 }
 
+// DISCUSS V3: where did the file method go?
```

Comment:
It's now in the assessor, and we also need to set other values, see https://github.com/centrifuge/tinlake/blob/develop/src/lender/assessor.sol#L83-L97

seniorInterestRate
maxReserve
maxSeniorRatio
minSeniorRatio